lib/active_shipping/shipment_packer.rb

```diff
@@ -20,7 +20,7 @@ def pack(items, dimensions, maximum_weight, currency)
 
         # Naive in that it assumes weight is equally distributed across all items
         # Should raise early enough in most cases
-        total_weight = calculate_total_weight(items, maximum_weight)
+        validate_total_weight(items, maximum_weight)
```

Comment:
I realized that we dont actually do anything with the accumulator.. we only use it to validate the total weight, changed the method name and no longer using `inject`